### PR TITLE
zh-trans: add container.md

### PR DESCRIPTION
**Pull request by AdamDang**
zh-trans: add container.md

### CODE REVIEW

File: content/zh/docs/reference/glossary/container.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+- fundamental
+- workload
+---
+-->
+
+<!--
+ A lightweight and portable executable image that contains software and all of its dependencies.
+-->
+
+ 容器是可移植和执行的轻量级的镜像，镜像中包含软件及其相关依赖。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/container.md]: here too

File: content/zh/docs/reference/glossary/container.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: 容器
+id: container
+date: 2018-04-12
+full_link: /docs/concepts/overview/what-is-kubernetes/#why-containers
+short_description: >
+  容器是可移植和执行的轻量级的镜像，镜像中包含软件及其相关依赖。
```
[the remark this answers is not in the document]
Comment (AdamDang) [reply content/zh/docs/reference/glossary/container.md]: OK

**Pull request merged**